falcon/constants.py

```diff
@@ -11,19 +11,26 @@
     'TRACE',
 )
 
-# RFC 3253 methods
+# RFC 2518 and 4918 methods
+
 WEBDAV_METHODS = (
     'CHECKIN',
     'CHECKOUT',
+    'COPY',
+    'LOCK',
+    'MKCOL',
+    'MOVE',
+    'PROPFIND',
+    'PROPPATCH',
     'REPORT',
     'UNCHECKIN',
+    'UNLOCK'
     'UPDATE',
     'VERSION-CONTROL',
 )
 
 COMBINED_METHODS = HTTP_METHODS + WEBDAV_METHODS
 
-
```

Comment:
Any specific reason for re-styling whitespace here?
(Or just trying to make a compelling case for `black` :wink: )